src/index.js

```diff
@@ -320,6 +321,12 @@ function initialize(env, context, specifiedOptions, platform, extraOptionDefs) {
     }
 
     if (sendEvent) {
+      // An event will be send for each of these by virtue of sending events for all flags.
+      if (!allFlags) {
```

Comment:
So, I am not 100% sure there is a good answer here. For sure if you are sending events for all flags, then the prereq will have an event if it was visible to the client.

Reply:
This comment is still relevant github. I just corrected part of the line. >.<